packages/trpc-interface/src/authorize/project.server.ts

```diff
@@ -157,6 +172,43 @@ export const hasProjectPermit = async (
   }
 };
 
+export const hasProjectPermit = async (
+  props: {
+    projectId: string;
+    permit: AuthPermit;
+  },
+  context: AppContext
+) => {
+  const start = Date.now();
+
+  try {
+    const { authorization } = context;
+
+    const authInfo: AuthInfo | undefined = authorization.isServiceCall
+      ? { type: "service" }
+      : authorization.authToken !== undefined
+        ? { type: "token", authToken: authorization.authToken }
+        : authorization.userId !== undefined
+          ? { type: "user", userId: authorization.userId }
+          : undefined;
+
+    if (authInfo === undefined) {
+      return false;
+    }
+
+    return checkProjectPermit(
+      props.projectId,
+      props.permit,
+      authInfo,
+      context.postgrest.client
+    );
+  } finally {
+    const diff = Date.now() - start;
+
+    console.info(`hasProjectPermit execution ${diff}ms`);
```

Comment:
why should this be logging all the time?

Reply:
Not the change of current PR

Reply:
its ok, just don't get this type of stuff in general, we shouldn't be spamming console on a regular basis without a reason or control, right?

Reply:
this could be also a debug statement I guess

Reply:
We used 3rd party proovider for this. Since that time it's not needed as no issues with permissions checks we have

Reply:
Removed